tests/travis/Couchbase.sh

```diff
@@ -7,8 +7,8 @@ docker run -d --name couchbase-server -p 8091:8091 -p 11210:11210 webpt/couchbas
 
 pecl uninstall couchbase
 
-sudo wget -O/etc/apt/sources.list.d/couchbase.list http://packages.couchbase.com/ubuntu/couchbase-ubuntu1204.list
+sudo wget -O/etc/apt/sources.list.d/couchbase.list http://packages.couchbase.com/ubuntu/couchbase-ubuntu1404.list
```

Comment:
xenial is not 14-04, but 16-04

Reply:
Changes were from me trying to get this to work again. I can drop these changes or remove the file entirely since we’re no longer calling it.

Reply:
I've removed the file entirely.